types/libav-core/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+    "npm-naming": false
```

Comment:
Please don't disable this lint rule - in this case, I think you've got the package name wrong. If this is for this package: https://www.npmjs.com/package/@ffmpeg/libav-core

You should write types for in a `ffmpeg__libav-core` folder and write `@ffmpeg/libav-core` as the package name in the header.

Reply:
Thanks for the suggestion, fixed with a new commit.